platform.md

```diff
@@ -746,27 +746,31 @@ Usage:
 | `<direct>`          |                       |                | Process execution strategy
 | `<cmd>`             |                       |                | Command to execute
 | `<args>`            |                       |                | Arguments to command
+| `<working-directory>`              |        |                | Process working directory
```

Comment:
```suggestion
```
Unless I am understanding this incorrectly `<working-directory>` is not a separate input the platform can provide to the launcher. Instead the working directory is either `<app>` or is read from `metadata.toml`.